### PR TITLE
Clean up styles/address accessibility issues

### PR DESCRIPTION
**Pull request by sayers24**
Related issue: #39 
 
- [ ]  WCAG 4.1.2 - buttons must have discernible text
- [ ]  WCAG 1.3.1, WCAG 3.3.2 - input controls must be associated with their labels
- [ ]  WCAG 1.4.3 - contrast ratio between text and background must meet AA standards
- [ ]  Keyboard navigation - tooltip information should be accessible for non-mouse users

### CODE REVIEW

File: covasim/webapp/index.html [hunk 1]
```diff
@@ -70,30 +70,30 @@ <h5 class="modal-title" id="exampleModalLabel">About Covasim</h5>
 
   <div class="content row">
 
-    <div id="parameters" class="col">
+    <div id="parameters" class="col-sm">
 
       <div class="row">
 
         <fieldset class="col-md-6">
```
Comment (gwincr11) [inline covasim/webapp/index.html:77]: I wonder if we can move these fieldsets to a partial, there are 2 so far that are the same and I have a pr open that starts a 3rd. https://012.vuejs.org/api/elements.html#partial

File: covasim/webapp/index.html [hunk 2]
```diff
@@ -102,64 +102,61 @@ <h5 class="modal-title" id="exampleModalLabel">About Covasim</h5>
...
           Run
-        </button>&nbsp;&nbsp;&nbsp;&nbsp;
-        <button id="reset_btn" :disabled="running" class="btn btn-primary blueish" type="button" @click="resetPars">
+        </button>
+        <button id="reset_btn" :disabled="running" class="btn btn-primary btn-lg ml-2 mr-2 blueish" type="button" @click="resetPars">
           Reset
-        </button>&nbsp;&nbsp;&nbsp;&nbsp;
-        <b-tooltip target="reset_btn" triggers="hover">
+        </button>
+        <b-tooltip target="reset_btn" triggers="hover focus">
```
Comment (gwincr11) [inline covasim/webapp/index.html:116]: Maybe we can add a tooltip partial, there are a bunch scattered around and it maybe nice to have a simple function that can build one?

Comment (gwincr11) [reply covasim/webapp/index.html:116]: I guess they call them components 😅 https://vuejs.org/v2/guide/#Composing-with-Components

File: covasim/webapp/index.html (hunk 1 above)
Comment (gwincr11) [reply covasim/webapp/index.html:77]: A component maybe better suited https://vuejs.org/v2/guide/#Composing-with-Components

**Comment by sayers24** [comment]
> Looks like a great cleanup. I wonder if we can use some templates to encourage consistency in our rendered elements?

Re: templates. Are we looking to keep the UI small or are we open to restructuring the web app with some tools to make future development faster/cleaner? (e.g. using vue-cli and component application file structure)

**Comment by cliffckerr** [comment]
@sayers24 @gwincr11 I think if we _can_ keep the UI small, that's preferable, especially if we want any hope of non-professional devs being able to fork and adapt it ("The CLI assumes prior knowledge of Node.js and the associated build tools." -- you've lost 95% of people right there). However, these tools exist for a very good reason, and I wouldn't want us to inconvenience ourselves by not using them. Let's _try_ to keep it simple, but then more to Vue-CLI if "simple" is no longer possible.

**Pull request merged**